Drop support for Ruby 2.3 (#101)
chef/kitchen-vcenter: README.md

## Requirements

- Ruby 2.4 or higher
- VMware vCenter/vSphere 5.5 or higher
- VMs or templates to clone, with open-vm-tools installed
- DHCP server to assign IPs to kitchen instances

## Installation

> This gem ships as part of Chef Workstation. See the [Chef Workstation Download Page](https://downloads.chef.io/products/workstation) for the install package.

Reviewer: ```suggestion
This gem ships as part of Chef Workstation. See the [Chef Workstation Download Page](https://downloads.chef.io/products/workstation) to download the installation package.
```
